Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -122,24 +151,60 @@ props.buildReportOrder.each{ buildReportFile ->
 		}
 	}
 
+    buildRecords += USSBuildRecords
+
 	def count = 0
+    def USSCount = 0	
```

Comment:
Same as in the dbb-ucd-packaging script.
`zosCount` & `ussCount`

Reply:
Done